server/Arcadia.Assistant.Calendar/SickLeave/EmployeeSickLeaveActor.cs

```diff
@@ -7,23 +7,25 @@
 
     using Arcadia.Assistant.Calendar.Abstractions;
     using Arcadia.Assistant.Calendar.SickLeave.Events;
+    using Arcadia.Assistant.Notifications.Abstractions;
+    using Arcadia.Assistant.Organization.Abstractions;
 
     public class EmployeeSickLeaveActor : CalendarEventsStorageBase
     {
-        private readonly IActorRef sickLeaveNotifications;
+        private readonly EmployeeMetadata employee;
```

Comment:
EmployeeMetadata can change, that's why it stored id only before

Reply:
Changed SickEmail actor notification contract to have only needed fields like employee name and start date. Then sending actor will be responsible to get the data.

Reply:
EmployeeSickLeaveActor might not have proper data, that's what I'm saying. If employee's name changes, or department. You either need to notify this actor about this changes (that's, by the way, possible) or request it somehow.

Reply:
Ok then, let's employee sick actor has reference to it's parent employee actor and gets data before send notification.